bin/slurm-send-mail.py

```diff
@@ -305,13 +359,20 @@ def runCommand(cmd):
 
 						# create HTML e-mail
 						msg = MIMEMultipart('alternative')
-						msg['subject'] = 'Job %s.%d: %s' % (cluster, jobId, state)
+						msg['subject'] = 'Job %s.%s: %s' % (cluster, job_name, state)
```

Comment:
I prefer the job ID to be in the subject line of the e-mail. However, I could add an option to `conf.d/slurm-mail.conf` that allows the format of the subject to be customisable, a bit like the date format can. I'll add an issue for this.

Reply:
Customisable e-mail subject now implemented in 1.7 branch.